src/Deposit.js

```diff
@@ -210,16 +293,26 @@ export default class Deposit {
         return new Deposit(factory, contract, keepContract)
     }
 
-    static async forTDT(factory/*: DepositFactory*/, tdt/*: TBTCDepositToken | string*/)/*: Promise<Deposit>*/ {
+    /**
+     * @param {DepositFactory} factory 
+     * @param {any | string} tdt
```

Comment:
Is it useful to have a sum type with `any` as a constituent? Forgive me if I'm misreading this

Reply:
This function is still unimplemented, but don't tell the project lead :wink:

Reply:
(If you look at the original type annotation, my intent is for this to be able to take an instance of `TBTCDepositToken`, but that's a wrapper class that doesn't exist yet.)

Reply:
Shwew, just checking my sanity 👌

Reply:
Yeah no, great callout. `any | string` is about as useful as an empty pizza box.